CMakeLists.txt

```diff
@@ -92,11 +92,10 @@ option(DISABLE_INTERNAL_OPENSSL "Disable internal OpensSSL binaries use" OFF)
 set(USING_INTERNAL_OPENSSL false)
 
 if(MSVC)
-  if((EXISTS "${PROJECT_SOURCE_DIR}\\openssl\\VS_120\\") AND NOT DISABLE_INTERNAL_OPENSSL)
+  if((EXISTS "${PROJECT_SOURCE_DIR}/openssl/VS_120/") AND NOT DISABLE_INTERNAL_OPENSSL)
```

Comment:
see my [comment](https://github.com/pocoproject/poco/pull/2120#issuecomment-361422177)

Reply:
Yes I know ... I merge the latest develop into my branch and changed the path in CMakeLists.txt.